packages/rocketchat-ui-master/server/inject.js

```diff
@@ -71,8 +71,7 @@ RocketChat.settings.get('Assets_SvgFavicon_Enable', (key, value) => {
 });
 
 RocketChat.settings.get('theme-color-sidebar-background', (key, value) => {
-	Inject.rawHead(key, `<style>body { background-color: ${ value };}</style>` +
```

Comment:
any reason removing it from here and adding it to "external" CSS? the idea was just this, not need to wait for the whole CSS to apply the initial background color

Reply:
When you load the app the css is up already, but I forgot that you can change this color... in that case the app will load the default color. I'll fix that.

Reply:
the first time you enter on a server, his css is not up already 😉

Reply:
I was wrong on my comment. @ggazzo made a dynamic css loader script that runs before that, so it will work with the colors that the user set on DB.